code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1233,4 +1233,26 @@ Traitors and the like can also be revived with the previous role mostly intact.
 /mob/living/carbon/proc/adminpie(mob/user)
 	var/obj/item/reagent_containers/food/snacks/pie/cream/p = new (get_turf(pick(oview(3,user))))
 	p.pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE | PASSBLOB | PASSCLOSEDTURF | LETPASSTHROW
-	p.throw_at(user, 10, 0.5, usr)
\ No newline at end of file
+	p.throw_at(user, 10, 0.5, usr)
+
+/client/proc/admincryo(mob/living/carbon/human/target as mob)
+	set category = "Admin"
+	set name = "Admin Cryo"
+	if(!check_rights(R_ADMIN))
+		return
+	var/offer = alert(usr, "Do you want to offer to ghosts?", "Ghost Offer", "Yes", "No")
+	if(offer == "Yes" && offer_control(target))
+		return
```

Comment:
Just a thing so that its clearer on what it actually does (try to offer then cryo if it cant)

```suggestion
	var/offer = alert(usr, "Do you want to try to offer to ghosts first?", "Ghost Offer", "Yes", "No")
	if(offer == "Yes" && offer_control(target))
		return
```